DOCS/man/vo.rst

```diff
@@ -28,10 +28,11 @@ Available video output drivers are:
 
     See `GPU renderer options`_ for options specific to this VO.
 
-    By default, it tries to use fast and fail-safe settings. Use the
-    ``gpu-hq`` profile to use this driver with defaults set to high quality
-    rendering. The profile can be applied with ``--profile=gpu-hq`` and its
-    contents can be viewed with ``--show-profile=gpu-hq``.
+    By default, mpv utilizes settings that balance quality and performance.
+    Two predefined profiles are available: ``fast`` for optimal performance and
```

Comment:
"Two additional predefined profiles" perhaps

Reply:
Technically "default" is not a profile

Reply:
"optimal" -> "maximum", since 'optimal' is subjective and (to me) implies some sort of balance between performance and quality trade-offs.